src/Microsoft.AspNetCore.Razor.Tools/project.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Razor.Design": "1.0.0-*",
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-003890",
```

Comment:
The preview3 specification will be removed entirely once we pull in the good bits/make sure we don't have rc2 ones on our feeds 😄.